contrib/scriptlets/deb/postinst

```diff
@@ -24,7 +24,10 @@ userModMessage="To allow other users to control the application run 'usermod -aG
 userDelMessage="To limit access exclusively to the root user, remove all users from the '$NORDVPN_GROUP' group."
 helpMessage="If you need help using the app, use the command 'nordvpn --help'."
 
-ldconfig # update shared library cache
+# update configuration and shared library cache for linker to find .so files
+echo "/usr/lib/nordvpn" > /etc/ld.so.conf.d/nordvpn.conf
+ldconfig
```

Comment:
shouldn't `ldconfig` be also executed at package uninstall?

Reply:
I'll check that :+1:

Reply:
If these libraries are supposed to be private for the package the global linker configuration should not be changed.

Reply:
The alternative is rpath, but the issue here is that rpath will be set only on our binaries, not on the dependencies, so for example `libtelio.so` won't find `libsqlite3.so`. I would have to move `libsqlite3.so` somewhere available for the linker risking overriding user's version, or I would have to put it somewhere not commonly available and modify the linker configuration anyway